.github/workflows/snap.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish Snap
+
+on:
+  # schedule:
+  #   - cron: "0 0 * * *" # Every day at 0:00
+  # workflow_dispatch: # or manually
+  push:
+    branches:
+      - realthunder-snap
+    
+jobs:
+  publish_amd64:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: install snapcraft
+      run: |
+        sudo snap install snapcraft --classic
+      #   echo "$STORE_LOGIN" | snapcraft login --with -
+      # env:
+      #   STORE_LOGIN: ${{ secrets.STORE_LOGIN }}
```

Comment:
You'd need to export a login token/macaroon from snapcraft and add it as a secret in GitHub, 
e.g. `snapcraft export-login --snaps=freecad-realthunder --channels=edge,candidate store-login`

Reply:
If you want to be able to push to stable directly, use `snapcraft export-login --snaps=freecad-realthunder --channels=edge,stable store-login`